CHANGELOG.md

```diff
@@ -12,11 +12,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
   the `meta.security` field of the corresponding FHIR resource will now be [appropriately populated][nopat-docs].
 * When the SNOMED DB ingest script has not completed successfully, The GP2GP Translator Service will now exit and throw a
   RuntimeException with the following message:
+  
   ```
   FATAL: Expected Immunization codes not found in snomedct.immunization_codes view.
   SNOMED CT Database not set up correctly.
-  Please update / reload the SNOMED DB.
-  ``` 
+  Please update / reload the SNOMED DB.  
+  ```
+
+* Added support for Organization reference types in ReferralRequest. 
```

Comment:
I'm pretty sure this wasn't part of release `3.0.1`

Reply:
oh, yeah, good point, it was  a wrongly resolved merge conflict

Reply:
fixed now